src/main/resources/static/css/style2.css

```
@@ -274,3 +280,156 @@ main{
+    padding: 20px 24px;
+    border-radius: 12px;
+
+    width: max-content;
+    min-width: 240px;
+    max-width: 320px;
+
+    box-shadow: 0 10px 25px rgba(0,0,0,0.2);
+    display: none;
+    z-index: 100;
+
+    opacity: 0;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)